devito/operators.py

```diff
@@ -31,9 +38,16 @@ def apply(self):
         f(*args)
         return tuple([param.data for param in self.output_params])
 
-    def get_callable(self):
+    def get_callable(self):           # each propagator passed here will add a function to generated cpp script
         self.jit_manager = JitManager([self.propagator], dtype=self.dtype, openmp=self.openmp)
-        return self.jit_manager.get_wrapped_functions()[0]
+        wrapped_function = self.jit_manager.get_wrapped_functions()[0]
+
+        if self.auto_tune and self.cache_blocking:
+            #                               current isat insall dir. Change based on your environment
+            at_controller = AtController("%s/isat" % os.getenv("HOME"))         # = space order
```

Comment:
We def can. Although I'm agains using env var for this and is creating config file just for one line necessary? If we're gonna be using it apart from this then yeah.